### DISCUSSION TITLE
cambio caldaia, che fare?

### DISCUSSION THREAD

**Discussion by hackaro** [indomus/forum#3260] [GENERALE - Discussioni generiche]
Buongiorno a tutti,
sono alle prime armi ma, poiché sto per cambiare la caldaia con una nuova a condensazione, ho pensato di dotarla di funzionalità domotiche. Vorrei prendere il kit Tado X, collegarlo al mio Apple HomePod mini che farebbe da Matter/Thread Border router e controllarlo da cellulare. Ho anche un Raspberry Pi-4 su cui installerò HA, ma in seguito. La domanda è molto semplice: non riesco a trovare un modello di caldaia a condensazione nella fascia 24-30kW (abito in un trilocale termo-autonomo e con necessità anche di acqua calda sanitaria) che supporti completamente il protocollo OpenTherm in modo da avere piena compatibilità con il termostato Tado X. Siete in grado di aiutarmi? Grazie anticipate!

**Comment by Starc73** [comment]
sul sito Tado vedi le caldaie compatibili
https://help.tado.com/it/articles/9263876-tado-x-e-compatibile-con-altre-linee-di-prodotti-tado
in particolare 
https://cdn.bfldr.com/607DGEMS/at/x6ss3p464trtbkpf428kj8c/IT_STx_compatibility_Links.pdf

**Comment by KC-inDomus** [comment]
> Buongiorno a tutti, sono alle prime armi ma, poiché sto per cambiare la caldaia con una nuova a condensazione, ho pensato di dotarla di funzionalità domotiche. Vorrei prendere il kit Tado X, collegarlo al mio Apple HomePod mini che farebbe da Matter/Thread Border router e controllarlo da cellulare. Ho anche un Raspberry Pi-4 su cui installerò HA, ma in seguito. La domanda è molto semplice: non riesco a trovare un modello di caldaia a condensazione nella fascia 24-30kW (abito in un trilocale termo-autonomo e con necessità anche di acqua calda sanitaria) che supporti completamente il protocollo OpenTherm in modo da avere piena compatibilità con il termostato Tado X. Siete in grado di aiutarmi? Grazie anticipate!

opentherm non è un obbligo. puoi serenamente attivare qualsiasi caldaia tradizionale con tado
